connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -18,6 +18,7 @@ export async function changeZendeskClientSubdomain({
   client: Client;
   brandId: number;
 }) {
+  // TODO: in some cases the brand in db is available and can be used to retrieve the subdomain
```

Comment:
`nit`: we usually add name and date on todays (e.g. `TODO (jules 10/10/2024): ...`)

Reply:
will do!